src/js/tech/tech.js

```diff
@@ -33,10 +34,20 @@ import document from 'global/document';
 class Tech extends Component {
 
   constructor(options={}, ready=function(){}){
+    let defaults = {
```

Comment:
Oh, ok, I see now. It's just the flag to turn these things off.
Can HLS turn off native audio tracks early enough for that to be sufficient for it's usecase?